HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MockupObjects.cs

```diff
@@ -197,18 +197,18 @@ public List<MapObject> getAllMapObjects()
 
 
 
-                double yOfObject = i == 0 ? startYOfDrawing : startYOfDrawing + heightOfObject * i - i * Constants.AllConstants.RECTANGLE_STROKE_THICKNESS;
+                double yOfObject = i == 0 ? startYOfDrawing : startYOfDrawing + heightOfObject * i - i * AllConstants.RECTANGLE_STROKE_THICKNESS;
 
 
                 if (i == 3)
                 {
-                    yOfObject += 80 + i * Constants.AllConstants.RECTANGLE_STROKE_THICKNESS;
+                    yOfObject += 80 + i * AllConstants.RECTANGLE_STROKE_THICKNESS;
                 }
 
```

Comment:
What is 'i', give it a more descriptive name if you use it in so much line of code.  Also, why you use 6? Can you make const for that? Why I propose const? Because if someone comes to code and looks, they will now why is 6.

Reply:
I actually kind of disagree with this, since this is a mockup class that's getting deleted relatively soon anyway. Tangling it up with constants will make it harder to delete, and trying too hard to make it clean is really just wasted effort.

Reply:
@milana-todorovic  Yes, I just used "i" as iterator to draw column of rooms. It doesn't have any special meaning and also each of this objects is saved separately to .txt file and will be loaded from it.

Reply:
I actually have one question, are you will keep this code for a possible move to Mockup data to test or you will delete it when you finish with it? 

  - If you have an idea to keep it in our code, then I suggest applying my proposal.
  - If you will delete it, and never again use it, that is okay, then make it like this

Reply:
@vlaksi  Okay. We only keep that function in case of some failure with .txt file (to fill the .txt with map objects). It's not constantly used, but just in case if something goes wrong. We will probably delete it in future, but for now on it will stay as a precaution...